homeassistant/components/switcher_kis/cover.py

```diff
@@ -62,11 +62,12 @@ class SwitcherCoverEntity(SwitcherEntity, CoverEntity):
         | CoverEntityFeature.SET_POSITION
         | CoverEntityFeature.STOP
     )
+    _cover_id: int
```

Comment:
This one is not needed, it can be deferred

Reply:
It is done.